ouroboros-network/src/Ouroboros/Network/Diffusion/Configuration.hs

```diff
@@ -0,0 +1,109 @@
+{-# LANGUAGE NamedFieldPuns #-}
+
+module Ouroboros.Network.Diffusion.Configuration
+  ( module Ouroboros.Network.Diffusion.Configuration
```

Comment:
It's better to explicitly export.  It would also be good to add haddock sections, some of them fall into different groups: `OutboundGovernor`, `InboundGovernor`, `Protocol params`.

Reply:
In general you're right, the reason is when I was writing it I came to the conclusion that this sort of configuration module consisting of defaults and constants is a special case where I imagine we probably will end up exporting everything anyhow. But I will change it to explicit export list.